editor/src/components/inspector/inspector-common.tsx

```diff
@@ -235,3 +240,26 @@ export function widthHeightFromAxis(axis: Axis): 'width' | 'height' {
       assertNever(axis)
   }
 }
+
+export function convertWidthToFlexGrow(
+  metadata: ElementInstanceMetadataMap,
+  elementPath: ElementPath,
+): Array<CanvasCommand> {
+  const element = MetadataUtils.getJSXElementFromMetadata(metadata, elementPath)
+  if (element == null) {
+    return []
+  }
+  const matches =
+    defaultEither(
+      null,
+      getSimpleAttributeAtPath(right(element.props), PP.create(['style', 'width'])),
+    ) === '100%'
+
+  if (!matches) {
```

Comment:
This should also be checking that the flex direction of the parent is horizontal, as we should doing the same thing with `height` if the direction is vertical